dev/EnvironmentManager/ChangeTracker/EnvironmentVariableChangeTrackerHelper.h

```diff
@@ -0,0 +1,60 @@
+﻿#pragma once
+#include <windows.h>
+#include <wil/result.h>
+#include <appmodel.h>
+#include <Microsoft.Windows.System.EnvironmentManager.h>
+
+using namespace winrt::Microsoft::Windows::System::implementation;
+
+static bool DoesChangeTrackingKeyExist()
+{
+    std::filesystem::path subKey = std::filesystem::path{ LR"(Software\Microsoft\AppModel\RegistryWriteVirtualization\ExcludedKeys\HKEY_CURRENT_USER/Software/ChangeTracker)" };
+    wil::unique_hkey keyToChangeTracker{};
+    LSTATUS openResult{ RegOpenKeyEx(HKEY_LOCAL_MACHINE, subKey.c_str(), 0, KEY_READ, keyToChangeTracker.put()) };
```

Comment:
`KEY_READ` should be `KEY_READ | KEY_WOW64_KEY` ?

Isn't this checking a _singular_ well defined registry location? 32bit apps will read the wrong location without `KEY_WOW64_KEY`?

See https://stackoverflow.com/questions/12796624/key-wow64-32key-and-key-wow64-64key for more info

Reply:
Yes correct.  I'll need to, at some point in time, consolidate all key calls so I don't have to duplicate `KEY_READ | KEY_WOW63_64KEY`

Good catch.